omc3/model/accelerators/sps/correctors.json

```diff
@@ -1,226 +1,226 @@
-        "kdh50807",
-        "kdh51007",
-        "kdh51207",
-        "kdh51407",
-        "kdha51606",
-        "kdhd51832",
-        "kdh52007",
-        "kdh52207",
-        "kdh52407",
-        "kdh52607",
-        "kdh52807",
-        "kdh53007",
-        "kdh53207",
-        "kdh53407",
-        "kdh53607",
-        "kdh60207",
-        "kdh60407",
-        "kdh60607",
-        "kdh60807",
-        "kdh61007",
-        "kdh61207",
-        "kdh61407",
-        "kdh61607",
-        "kdhb61805",
-        "kdh62007",
-        "kdh62207",
-        "kdh62407",
-        "kdh62607",
-        "kdh62807",
-        "kdh63007",
-        "kdh63207",
-        "kdh63407",
-        "kdh63607",
-        "kdv10107",
-        "kdv10307",
-        "kdv10507",
-        "kdv10707",
-        "kdv10907",
-        "kdv11107",
-        "kdv11307",
-        "kdv11507",
-        "kdv11705",
-        "kdva11904",
-        "kdv12107",
-        "kdv12307",
-        "kdv12507",
-        "kdv12707",
-        "kdv12907",
-        "kdv13107",
-        "kdv13307",
-        "kdv13507",
-        "kdv20107",
-        "kdv20307",
-        "kdv20507",
-        "kdv20707",
-        "kdv20907",
-        "kdv21107",
-        "kdv21307",
-        "kdv21507",
-        "kdva21703",
-        "kdva21932",
-        "kdv22107",
-        "kdv22307",
-        "kdv22507",
-        "kdv22707",
-        "kdv22907",
-        "kdv23107",
-        "kdv23307",
-        "kdv23507",
-        "kdv30107",
-        "kdv30307",
-        "kdv30507",
-        "kdv30707",
-        "kdv30907",
-        "kdv31107",
-        "kdv31307",
-        "kdv31507",
-        "kdv31707",
-        "kdv31907",
-        "kdv32107",
-        "kdv32307",
-        "kdv32507",
-        "kdv32707",
-        "kdv32907",
-        "kdv33107",
-        "kdv33307",
-        "kdv33507",
-        "kdv40107",
-        "kdv40307",
-        "kdv40507",
-        "kdv40707",
-        "kdv40907",
-        "kdv41107",
-        "kdv41307",
-        "kdv41507",
-        "kdva41703",
-        "kdva41932",
-        "kdv42107",
-        "kdv42307",
-        "kdv42507",
-        "kdv42707",
-        "kdv42907",
-        "kdv43107",
-        "kdv43307",
-        "kdv43507",
-        "kdv50107",
-        "kdv50307",
-        "kdv50507",
-        "kdv50707",
-        "kdv50907",
-        "kdv51107",
-        "kdv51307",
-        "kdv51507",
-        "kdv51707",
-        "kdv51920",
-        "kdv52107",
-        "kdv52307",
-        "kdv52507",
-        "kdv52707",
-        "kdv52907",
-        "kdv53107",
-        "kdv53307",
-        "kdv53507",
-        "kdv60107",
-        "kdv60307",
-        "kdv60507",
-        "kdv60707",
-        "kdv60907",
-        "kdv61107",
-        "kdv61307",
-        "kdv61507",
-        "kdva61703",
-        "kdva61932",
-        "kdv62107",
-        "kdv62307",
-        "kdv62507",
-        "kdv62707",
-        "kdv62907",
-        "kdv63107",
-        "kdv63307",
-        "kdv63507"
+        "kmdh10207",
```

Comment:
Trusting you on these ones. Can you just let me know where you got the updated info from? Directly lattice files?

Reply:
"Updated" is wrong. this file is new. The first info was more a guess, based on LHC nomenklature, the change was because I checked the sequence.

Reply:
ok, thanks